spec/controllers/scim_rails/scim_groups_controller_spec.rb

```diff
@@ -687,6 +688,71 @@
           expect(updated_user_list.length).to eq(1)
           expect(updated_user_ids).to include(new_user_id)
         end
+
+        context "with azure requests" do
+          context "when Replace operation" do
+            let(:new_name) { Faker::Name.name }
+
+            it "processes path and updates attribute" do
+              patch :patch_update, params: azure_patch_params(id: target_group.id, operation: "Replace", path: "displayName", value: new_name)
+
+              expect(updated_group.display_name).to eq(new_name)
+            end
+
+            it "processes path and activates group" do
+              patch :patch_update, params: azure_patch_params(id: target_group.id, operation: "Replace", path: "active", value: true)
```

Comment:
🌵 🌵 you could set up these requests to be in a let or subject to reduce the repetition
Then have separate `let` values for the parts that change